src/fullnode.rs

```diff
@@ -363,10 +363,9 @@ impl Fullnode {
                 }
                 None => FullnodeReturnType::LeaderToLeaderRotation, // value doesn't matter here...
             };
-            let mut new: Arc<Bank> = Arc::new(Bank::new_from_parent(self.bank.clone()));
-            std::mem::swap(&mut self.bank, &mut new);
+            let tpu_bank = Arc::new(Bank::new_from_parent(self.bank.clone()));
```

Comment:
Until the bank traverses parents to implement all its methods, that clone() should be within `new_from_parent()`.